examples/table/table.bal

```diff
@@ -74,24 +74,23 @@ public function main() {
 
     // `.map()` applies a function to each member of a table and returns a table of the result.
     // The resulting table will have the same keys as the argument table.
-    table<Person> personTab = employeeTab.'map(function (Employee employee)
-    returns Person {
-        return { id: employee.id, name:employee.name, age:23 };
-    });
+    table<Person> personTab = employeeTab.'map(function (Employee employee) returns Person {
+                                    return {id: employee.id, name:employee.name, age:23};
+                              });
```

Comment:
Btw, I didn't mean these two lines. These were correctly formatted previously. Only the `returns` part had to be formatted

Reply:
Check now